<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1516</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>10</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLineEdit" name="selected_file_name_line_edit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="file_selection_button">
       <property name="text">
        <string>(CHOOSE FILE)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="move_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>480</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MOVE</string>
    </property>
   </widget>
   <widget class="QLabel" name="move_x_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>510</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="move_x_up_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>570</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_move_line_edit">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>540</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="move_x_down_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotate_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>480</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ROTATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="x_rotate_label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>510</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_rotate_line_edit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>540</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_rotate_line_edit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>640</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="y_rotate_label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>610</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="z_rotate_line_edit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>750</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="z_rotate_label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>720</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="move_y_down_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>670</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="y_move_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>610</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_move_line_edit">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>640</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="move_y_up_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>670</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="move_z_down_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>780</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="z_move_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>720</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="z_move_line_edit">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>750</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="move_z_up_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>780</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+1</string>
    </property>
   </widget>
   <widget class="QLabel" name="scale_label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>610</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SCALE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="scale_line_edit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>660</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scale_down_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>650</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scale_up_button">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>650</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="objfile_name_label">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>480</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>OBJECT FILE NAME</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="objfile_name_line_edit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>540</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="vertices_amount_label">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>600</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>VERTICES AMOUNT</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vertices_amount_line_edit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>660</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="edges_amount_label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>720</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>EDGES AMOUNT</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edges_amount_line_edt">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>780</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="projection_type_label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>480</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PROJECTION TYPE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="projection_type_combo_box">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>530</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>parallel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>central</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="edges_width_label">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>480</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>EDGES WIDTH</string>
    </property>
   </widget>
   <widget class="QSlider" name="edges_width_slider">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>540</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="edges_colour_label">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>600</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>EDGES COLOUR</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="edges_red_spin">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>660</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="edges_green_spin">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>660</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="edges_blue_spin">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>660</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="red_colour_label">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>630</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QLabel" name="green_colour_label">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>630</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QLabel" name="blue_colour_label">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>630</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="edges_type_label">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>720</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>EDGES TYPE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="edges_type_combo_box">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>770</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>solid</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dashed</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="vertices_red_spin">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>660</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="vertices_size_label">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>480</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>VERTICES SIZE</string>
    </property>
   </widget>
   <widget class="QSlider" name="vertices_size_slider">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>540</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="vertices_display_method_combo_box">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>770</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>none</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>square</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="vertices_display_method">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>720</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>VERTICES DISPLAY METHOD</string>
    </property>
   </widget>
   <widget class="QLabel" name="green_colour_label_2">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>630</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QLabel" name="red_colour_label_2">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>630</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QLabel" name="blue_colour_label_2">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>630</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="vertices_green_spin">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>660</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="vertices_blue_spin">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>660</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="vertices_colour_label">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>600</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>VERTICES COLOUR</string>
    </property>
   </widget>
   <widget class="QLabel" name="background_colour_label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>720</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BACKGROUND COLOUR</string>
    </property>
   </widget>
   <widget class="QLabel" name="blue_colour_label_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>750</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="bg_green_spin">
    <property name="geometry">
     <rect>
      <x>571</x>
      <y>780</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="bg_red_spin">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>780</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="bg_blue_spin">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>780</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="red_colour_label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>750</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QLabel" name="green_colour_label_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>750</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gif_button">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>190</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>GIF</string>
    </property>
   </widget>
   <widget class="MyGLWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1231</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>270</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/avia/aviasales-vector-logo-small.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="error_line_edit">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>140</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple Braille</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="rotation_x_down_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>570</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotation_x_up_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>570</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotation_y_up_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>670</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotation_y_down_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>670</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotation_z_down_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>780</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotation_z_up_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>780</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="screen_button">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>230</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>SCREEN</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1516</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">myglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
